src/vrapp/VRVolumeApp.cpp

```diff
@@ -946,18 +978,17 @@ bool VRVolumeApp::data_is_multi_channel()
     for (int i = 0; i < m_volumes.size(); i++)
     {
       if (m_volumes.size() > 0 && m_volumes[i][0]->get_channels() > 1 &&
-        (renderchannel == 0 || renderchannel == 5 || renderchannel == 6))
+          (renderchannel == 0 || renderchannel == 5 || renderchannel == 6))
```

Comment:
what's special about these channels? maybe add a comment?

Reply:
Added a comment on the meaning of `renderchannel`